gentest/queue.sh

```diff
@@ -1,7 +1,12 @@
 #!/bin/bash
 test -f "$1" || exit
-QUEUE=https://sqs.ap-southeast-1.amazonaws.com/182387550209/invites
+# This is a hardcoded variable <-- should be moved
+QUEUE=https://sqs.ap-southeast-1.amazonaws.com/812644853088/invites
+#QUEUE=https://sqs.ap-southeast-1.amazonaws.com/915001051872/invites
+# END This is a hardcoded variable
 echo Attempting to put $1 onto $QUEUE
-aws --profile ins-dev sqs send-message-batch \
+# This is a hardcoded variable <-- should be moved
+aws --profile uneet-dev sqs send-message-batch \
```

Comment:
the profile information here seems to be incorrect. It should be `ins-dev` (still hardcoded for now).

Reply:
updated...